<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidgetGeneral</class>
 <widget class="QWidget" name="SettingsWidgetGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>452</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="rememberLastDatabasesCheckBox">
     <property name="text">
      <string>Remember last databases</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="rememberLastKeyFilesCheckBox">
     <property name="text">
      <string>Remember last key files</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="openPreviousDatabasesOnStartupCheckBox">
     <property name="text">
      <string>Load previous databases on startup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="autoSaveOnExitCheckBox">
     <property name="text">
      <string>Automatically save on exit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="autoSaveAfterEveryChangeCheckBox">
     <property name="text">
      <string>Automatically save after every change</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="autoReloadOnChangeCheckBox">
     <property name="text">
      <string>Automatically reload the database when modified externally</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="minimizeOnCopyCheckBox">
     <property name="text">
      <string>Minimize when copying to clipboard</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="useGroupIconOnEntryCreationCheckBox">
     <property name="text">
      <string>Use group icon on entry creation</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="autoTypeShortcutLabel">
     <property name="text">
      <string>Global Auto-Type shortcut</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="ShortcutWidget" name="autoTypeShortcutWidget"/>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="autoTypeEntryTitleMatchCheckBox">
     <property name="text">
      <string>Use entry title to match windows for global auto-type</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="languageLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="languageComboBox"/>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="systray1Layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="systrayMinimizeToTrayCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Hide window to system tray when minimized</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="systray2Layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="systrayMinimizeOnCloseCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Hide window to system tray instead of app exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <layout class="QHBoxLayout" name="systray3Layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="systrayMinimizeOnStartup">
       <property name="text">
        <string>Minimize window at application startup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="systrayShowCheckBox">
     <property name="text">
      <string>Show a system tray icon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutWidget</class>
   <extends>QLineEdit</extends>
   <header>autotype/ShortcutWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rememberLastDatabasesCheckBox</tabstop>
  <tabstop>rememberLastKeyFilesCheckBox</tabstop>
  <tabstop>openPreviousDatabasesOnStartupCheckBox</tabstop>
  <tabstop>autoSaveOnExitCheckBox</tabstop>
  <tabstop>autoSaveAfterEveryChangeCheckBox</tabstop>
  <tabstop>minimizeOnCopyCheckBox</tabstop>
  <tabstop>autoTypeShortcutWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
